schemas/README.md

```diff
@@ -2335,7 +2333,52 @@ bytes
 </td>
 <td>
 
-Raw image data
+Raw image data.
+
+The requirements for each `encoding` value are:
+- `yuv422`,  `uyvy`:
```

Comment:
Are these (and other similar pairs) synonyms? It's not clear to me why we have two field values with the same description.